### DISCUSSION TITLE
Welcome to Loaders Discussions!

### DISCUSSION THREAD

**Discussion by bkerler** [bkerler/Loaders#69] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mansaga** [comment]
Hi,
I'm searching the firehose file of the Fairphone 4 to unbrick my phone. Is there any file that is compatible with this phone? Thank you.

**Reply by ACN-C** [reply]
Using xbl.elf or xbl.img (from official OTA payload and other  OSs') seems to get some response.
I'm trying to see what else I can do with that.
Unfortunately,  the USB disconnects when trying to actually run something.

**Reply by mansaga** [reply]
I am not very sure that this will help with FP4. I think that was possible with some other devices, but for the FP4, I think there is some signature of the owners ...

**Comment by gusdlm** [comment]
Hi, I'm looking for the firehose loader file for a Meta Oculus Quest 2.

**Reply by ender9492** [reply]
Would love to find the loaders for all Oculus/Meta Quest headsets!

**Reply by SerraraFluttershy** [reply]
Yeah, same.
I don't think it's anywhere on the public internet.

**Reply by gxshao** [reply]
any luck?

**Comment by Sazendrak** [comment]
does this repo already have the proper loader for the one plus 6t? and are the engineering bootloaders for the lg v60 and v35 relevant contributions, i noticed you can’t access them anymore over on xda.

**Reply by bkerler** [reply]
I think they should be already included in the edl repo, but feel free to send a pull request with the loaders :)

**Comment by yissusluna82-design** [comment]
hola estoy buscando el cargador para flaseho en modo edl a
 un dispositivo honor x7b clk-lx3 con procesador snapdragon 680
